### PR TITLE
Milestone rebound

### PR DESCRIPTION
**Pull request by g4s8**
#609 - milestone rebound in `TkGithub`

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#797](https://github.com/zerocracy/farm/issues/797) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#797](https://github.com/zerocracy/farm/issues/797) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/711657332). The budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4). Please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket. If you decide that this PR should not be accepted ever, also inform the architect.

File: src/test/java/com/zerocracy/radars/github/RbMilestoneTest.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                        "milestone",
+                        Json.createObjectBuilder()
+                            .add("title", "milestone-title")
+                            .add("number", 1)
+                            .add("description", "some descripton")
+                            .add("due_on", "2018-04-06T07:00:00Z")
+                            .build()
+                    ).build()
+            ),
+            Matchers.startsWith("Milestone submitted")
```
Comment (carlosmiranda) [inline src/test/java/com/zerocracy/radars/github/RbMilestoneTest.java]: @g4s8 let's check the full message here since we know the milestone number.

**Comment by g4s8** [comment]
@carlosmiranda fixed in 9d215a8

**Comment by carlosmiranda** [comment]
@yegor256 good to merge

File: src/main/java/com/zerocracy/radars/github/TkGithub.java [hunk 2]
```diff
@@ -99,6 +99,7 @@ public TkGithub(final Farm frm) throws IOException {
                                 ),
                                 "created"
                             ),
+                            new RbByActions(new RbMilestone()),
```
Comment (yegor256) [inline src/main/java/com/zerocracy/radars/github/TkGithub.java]: @g4s8 you need a list of actions here, I believe :)

**Comment by g4s8** [comment]
@yegor256 sure) missed that. Fixed in e7fae00

**Comment by yegor256** [comment]
@rultor merge

**Comment by carlosmiranda** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/zerocracy/farm/issues/797#issuecomment-372881253))

@carlosmiranda This is what I know about this job, as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#797](https://github.com/zerocracy/farm/issues/797) is [in scope](http://www.zerocracy.com/policy.html#14) for 2days
  * The role is `REV`
  * The job is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda) for [2days](http://www.zerocracy.com/policy.html#8)
  * There is a monetary reward attached
  * @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda) will get 25% of their hourly rate for this job
  * Some equity will be [vested](http://www.zerocracy.com/policy.html#37) on completion
  * The job doesn't have any [impediments](http://www.zerocracy.com/policy.html#9)
  * The [budget](http://www.zerocracy.com/policy.html#4) is 15 minutes/points
  * These users are banned and won't be assigned:
    * @g4s8[/z](https://www.0crat.com/u/g4s8): This user reported the ticket
  * Job [footprint](https://www.0crat.com/footprint/C3NDPUA8L?q=%7Bjob%3A%27gh%3Azerocracy%2Ffarm%23797%27%7D) (restricted area)

**Comment by carlosmiranda** [comment]
@yegor256 need to close this order

**Comment by yegor256** [comment]
@0crat out

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/zerocracy/farm/issues/797#issuecomment-373188171))

@yegor256 The job [#797](https://github.com/zerocracy/farm/issues/797) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 points just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256)

**Comment by ypshenychka** [comment]
@carlosmiranda According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Onlye one issue was found during code review. 
Please confirm that you'll try to find at least three problems while future reviews.

**Comment by carlosmiranda** [comment]
@ypshenychka confirmed

**Comment by ypshenychka** [comment]
@carlosmiranda Thanks!

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
> @0crat quality acceptable ([here](https://github.com/zerocracy/farm/issues/797#issuecomment-373301355))

@ypshenychka Thanks, but quality review is not required in this job

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 points just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)

**Comment by 0crat** [comment]
Quality review completed: +8 points just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**